apps/services/handlers.py

```diff
@@ -73,7 +73,7 @@ async def create_user_request(
             session: AsyncSession
     ):
         user_faculty_result = await user_faculty_service.read(data={"user_id": user.user_id}, session=session)
-        data = {"date_created": datetime.now(),
+        data = {"created_at": datetime.now(),
```

Comment:
re-test please and check that datetime is saved in correct format

Reply:
Fixed and tested